vagrant/config/boxes.ci.yaml

```diff
@@ -0,0 +1,17 @@
+operating_systems:
```

Comment:
the CI doesn't use predefined boxes, only `versions.yaml` and then generates boxes based on that on the fly. so we don't need that file/profile at all (IMHO)